charts/puris/values.yaml

```diff
@@ -446,7 +452,11 @@ postgresql:
   auth:
     # -- Name of the database.
     database: "postgres"
-    # -- Username for the database.
+    # -- Secret containing the password. For more information look into 'backend-secrets-postgres.yaml' file.
+    existingSecret: "secret-postgres-init"
+    # -- Username for the root user and for the database.
```

Comment:
Not sure why you call it root user, it's a custom user creation as per docs: https://github.com/bitnami/charts/blob/main/bitnami/postgresql/README.md

<img width="777" alt="image" src="https://github.com/eclipse-tractusx/puris/assets/7613143/a28df062-f6c8-42b4-b679-a82ccd6609fa">

same two lines below for password.

Reply:
updated, thanks for pointing out